### PR TITLE
Fixes #648 - Export allows parameter mapping

### LINKED ISSUE #648: Export should allow parameters mapping

### PR DESCRIPTION
**Pull request by bradnussbaum**

### CODE REVIEW

File: src/main/java/apoc/export/csv/ExportCSV.java [hunk 1]
```diff
@@ -64,7 +66,8 @@ public ExportCSV() {
     @Procedure
     @Description("apoc.export.csv.query(query,file,config) - exports results from the cypher statement as csv to the provided file")
     public Stream<ProgressInfo> query(@Name("query") String query, @Name("file") String fileName, @Name("config") Map<String, Object> config) throws Exception {
-        Result result = db.execute(query);
+        if (config == null) config = Collections.emptyMap();
+        Result result = db.execute(withParamMapping(query, config.keySet()), config);
```
Comment (jexp) [inline src/main/java/apoc/export/csv/ExportCSV.java]: @bradnussbaum the config is not meant to contain the parameters directly, it's originally meant to have information about batching and file/export. Can you use an "parameters" param in config instead and then also add it to the docs + test. Thanks

Comment (bradnussbaum) [reply src/main/java/apoc/export/csv/ExportCSV.java]: @jexp Yes I can change to that.

**Comment by jexp** [comment]
Ping @bradnussbaum

**Comment by jexp** [comment]
Already in, thanks for the pointer.

**Pull request closed without merging**